CMakeLists.txt

```diff
@@ -0,0 +1,51 @@
+# Copyright Airtime Media 2021
+project(at_zookeeper_c)
+cmake_minimum_required(VERSION 3.12.0)
+
```

Comment:
Move this file to an `airtime` subdirectory

Reply:
Moved